traffic_monitor/conf/traffic_monitor.cfg

```diff
@@ -5,6 +5,7 @@
 	"http_timeout_ms": 2000,
 	"peer_polling_interval_ms": 5000,
 	"peer_optimistic": true,
+	"peer_optimisitc_quorum_min": 0,
```

Comment:
Any reason not to add this config setting to [the CDN-in-a-Box TM config](https://github.com/apache/trafficcontrol/blob/RELEASE-4.0.0-RC1/infrastructure/cdn-in-a-box/traffic_monitor/traffic_monitor.cfg)?

Reply:
For posterity we should, however, the feature will not work unless there are a minimum of three Traffic Monitors. I'll add it to the PR.